src/CreeDictionary/CreeDictionary/templates/CreeDictionary/components/search-result.html

```diff
@@ -65,7 +65,7 @@
       {% endfor %}
     </ol>
 
-    {% include "CreeDictionary/components/preverb-breakdown.html" %}
```

Comment:
If `lexical-info.html` is an updated and renamed version of `preverb-breakdown.html`, can you `git rm` the original file so that the diff for `lexical-info.html` shows changes instead of appearing as a brand-new file?

Reply:
It started as an entirely new file for reduplication only, then I renamed it when I decided to do preverbs and reduplication together. I don't think `git rm` will realize that?

Reply:
If we want to make it future proof, if only for crk, we should include prenouns as well. There in Arok's CW; I just haven't incorporated them in the FST yet.

Reply:
@nienna73 If the files have similar enough content, git can still infer that the removal + addition is a rename with changes.